manifests/s/SonicWALL/NetExtender/10.2.315/SonicWALL.NetExtender.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SonicWALL.NetExtender
+PackageVersion: 10.2.315
+InstallerSwitches:
+  Silent: /quiet /norestart
+  SilentWithProgress: /passive /norestart

```

Comment:
```suggestion
  SilentWithProgress: /passive /norestart
InstallerSuccessCodes: 
- 3010
```

Reply:
Unfortunately, when I add that `winget validate --manifest` fails with:

    Manifest validation failed.
    Manifest Error: Duplicate installer return code found.

I assumed 3010 was one of the ["known default values by winget"](https://github.com/microsoft/winget-cli/blob/4968df49d5ee25565fa9c37719e976c06787d97a/schemas/JSON/manifests/v1.0.0/manifest.singleton.1.0.0.json#L160).  Should I open an issue about `winget validate`?

Reply:
It's a bug in the latest version of WinGet: https://github.com/ItzLevvie/winget-pkgs-validate-and-install/issues/2#issuecomment-988243555

It's fixed in 1.2 and above.

Reply:
Great, thanks @ItzLevvie and @Trenly!